prometheus_pgbouncer_exporter/collector.py

```diff
@@ -99,10 +99,21 @@ def collect(self):
                     {"type": "gauge", "column": "pool_size",           "metric": "database_pool_size",           "help": "Configured Pool Size Limit"},
                     {"type": "gauge", "column": "reserve_pool",        "metric": "database_reserve_pool_size",   "help": "Configured Reserve Limit"},
                     {"type": "gauge", "column": "current_connections", "metric": "database_current_connections", "help": "Database connection count"},
+                    {"type": "gauge", "column": "max_connections",     "metric": "database_max_connections",   "help": "Database maximum number of allowed connections"},                    
```

Comment:
Please, align the `"help"` element with the one above it

```suggestion
                    {"type": "gauge", "column": "max_connections",     "metric": "database_max_connections",     "help": "Database maximum number of allowed connections"},                    
```

Reply:
Thank you for answering so fast. 
I have added a new commit solving the code issues you raised. And some extra fixes I raised would doing some integration testing.